### PR TITLE
BDCOM: add transceiver monitoring

### PR DESCRIPTION
**Pull request by freddy36**
Adds sensors for various transceiver parameters for BDCOM devices

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [X] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [X] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [X] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by murrant** [comment]
Darn bad timing.  Can you update this after #16165 is merged?

**Comment by freddy36** [comment]
Of course I'll update it

**Comment by freddy36** [comment]
@murrant If you're not planning to merge #16165 anytime soon we would appreciate it, if this PR could be merged.

File: mibs/bdcom/NMS-IF-MIB.my [hunk 1]
```diff
@@ -0,0 +1,1014 @@
+NMS-IF-MIB  DEFINITIONS ::= BEGIN
```
Comment (PipoCanaja) [inline mibs/bdcom/NMS-IF-MIB.my:1]: Could you please rename the file to NMS-IF-MIB (remove extension) ?

**Review by PipoCanaja — CHANGES_REQUESTED** [review]

File: mibs/bdcom/NMS-IF-MIB.my (hunk 1 above)
Comment (PipoCanaja) [reply mibs/bdcom/NMS-IF-MIB.my:1]: Other that this, looks good @freddy36

Comment (freddy36) [reply mibs/bdcom/NMS-IF-MIB.my]: fixed

**Pull request merged**